frontend/src/features/properties/list/PropertyListView.test.tsx

```diff
@@ -141,6 +141,71 @@ describe('Property list view', () => {
     });
   });
 
+  it('Displays edit button', async () => {
+    mockedService.getPropertyList.mockResolvedValueOnce({
```

Comment:
could reuse these mocks with a function or something to cut down on duplication, optional.